src/Operation/Aggregate.php

```diff
@@ -196,6 +204,10 @@ public function __construct($databaseName, $collectionName, array $pipeline, arr
             throw InvalidArgumentException::invalidType('"hint" option', $options['hint'], 'string or array or object');
         }
 
+        if (isset($options['let']) && ! is_array($options['let']) && ! is_object($options['let'])) {
+            throw InvalidArgumentException::invalidType('"let" option', $options['let'], 'array or object');
```

Comment:
`invalidType` supports an array of accepted types but I haven't managed to change all of these.

```suggestion
            throw InvalidArgumentException::invalidType('"let" option', $options['let'], ['array', 'object']);
```

Reply:
Ah, you added this back in https://github.com/mongodb/mongo-php-library/commit/b2cd2866c64690603896b788cd56e45ff4cc69a4. Totally forgot about that.